week-1/1-exercises/K-functions-parameters/exercise.js

```diff
@@ -1,9 +1,7 @@
-// Complete the function so that it takes input parameters
+// multiplication function//
+
 function multiply() {
-  // Calculate the result of the function and return it
+  return 3 * 4;
```

Comment:
Whilst this does work for the result below, it's not going to work for other numbers.  The Exercise asked for this function to work with input parameters to multiple together.